app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/zetime/ZeTimeDeviceSupport.java

```diff
@@ -70,7 +76,7 @@
     private final GBDeviceEventBatteryInfo batteryCmd = new GBDeviceEventBatteryInfo();
     private final GBDeviceEventVersionInfo versionCmd = new GBDeviceEventVersionInfo();
     private final GBDeviceEventMusicControl musicCmd = new GBDeviceEventMusicControl();
-    private final int sixHourOffset = 21600;
+    private final int sevenHourOffset = 25200;
```

Comment:
Might your change from sixHourOffset to sevenHourOffset have something to do with dayllight savings?

Reply:
I think the firmware of the watch was changed and gives us timestamps which are UTC -7h, now.  My timezone is UTC+1h. The watch shows correct time, but the timestamps reported are not right.

Reply:
We will soon see if it is related to DST :-)